### DISCUSSION TITLE
Force play album versions of songs, not the music video version.

### DISCUSSION THREAD

**Discussion by SprocketRocket23** [pear-devs/pear-desktop#1419] [Q&A]
Hey is there anyway to make it force play the album version of songs? Sometimes the music video has annoying end bumpers that doesn't get filtered out by sponserblock. Really breaks of the flow of playing a full album without shuffle.

**Comment by skypeias** [comment]
its a premium (server-side) feature so no there isn't a way to play the album version directly from the album page. the only workaround is to create a playlist of that album, simply by going to the menu, click add to playlist, create a new one for the album, and there, the album versions of the songs show up in the playlist.
